src/kv/error.rs

```diff
@@ -67,10 +67,4 @@ mod std_support {
             Error::boxed(err)
         }
     }
-
-    impl From<Error> for io::Error {
```

Comment:
We removed a similar external `From` impl in #398 that was causing problems with inference.